internal/scripts/installation/compile_MET_all.sh

```diff
@@ -469,17 +486,33 @@ if [ $COMPILE_G2CLIB -eq 1 ]; then
   rm -rf ${LIB_DIR}/g2clib/g2clib*
   tar -xf ${TAR_DIR}/g2clib*.tar -C ${LIB_DIR}/g2clib
   cd ${LIB_DIR}/g2clib/g2clib*
-  # Sed commands use double-quotes to support variable expansion.
-  sed -i "s|INC=.*|INC=-I${LIB_DIR}/include -I${LIB_DIR}/include/jasper|g" makefile
+  
+  if [[ $machine == "Mac" ]]; then
```

Comment:
There is a lot of duplication here. I think this could be simplified by creating a variable to store the the sed command to use based on the OS.

After the case statement (around line 356):

```
if [[ $machine == "Mac" ]]; then
    sed_inline="sed -i ''"
else
    sed_inline="sed -i''"
fi
```

then wherever you need to run a sed command, use the variable instead of "sed" e.g.

Before:
`sed -i '' "s|INC=.*|INC=-I${LIB_DIR}/include -I${LIB_DIR}/include/jasper|g" makefile`
After:
`$sed_inline "s|INC=.*|INC=-I${LIB_DIR}/include -I${LIB_DIR}/include/jasper|g" makefile`

Reply:
Thanks @georgemccabe for this simplification!  I'll change the code and retest to ensure the functionality still works.  I'll follow up.